<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoInterface</class>
 <widget class="QWidget" name="VideoInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="wid_fri" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="QLabel" name="lb_up">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wid_usr" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <widget class="QLabel" name="lb_down">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
